### PR TITLE
fix(one-off): remove async one-off 

### PR DESCRIPTION
**Pull request by SCedricThomas**
Related to https://github.com/Scalingo/app-scheduler/issues/886

This PR aims to remove the notion of async containers from the CLI as they are by default async  

- [x] Add a changelog entry in the section "To Be Released" of CHANGELOG.md

### CODE REVIEW

**Comment by SCedricThomas** [comment]
I don't know how I can test this in dev.  
Would you have any idea @ipfaze?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ### To be Released
 
+* fix(one-off): remove async field from the run command ([PR#1060](https://github.com/Scalingo/cli/pull/1060))
```
Comment (ipfaze) [inline CHANGELOG.md:5]: issue(blocking): I'm not sure we removed the async one-off feature here. But we did remove the `async` params as it is the default behavior.

**Review by ipfaze — CHANGES_REQUESTED** [review]
issue(blocking): I'm not sure we removed the async one-off feature here. But we did remove the `async` params as it is the default behavior.

File: CHANGELOG.md (hunk 1 above)
Comment (SCedricThomas) [reply CHANGELOG.md:5]: Yeah :/

Comment (SCedricThomas) [reply CHANGELOG.md:5]: Is it not clear enough?

Comment (ipfaze) [reply CHANGELOG.md:5]: My bad, I missed the `field` word 🤦
It's okay!

**Review by ipfaze — APPROVED** [review]
LGTM 👍

**Pull request merged**